llama_stack/cli/llama.py

```diff
@@ -8,6 +8,7 @@
 
 from .download import Download
 from .model import ModelParser
+from .shield import ShieldParser
```

Comment:
@ashwinb @leseb we have the `llama model|stack|download|verify-download` cli from the llama-stack repo and the `llama-stack-client datasets|eval|eval_tasks|inference|inspect|models|post_training|providers|scoring_functions|shields|toolgroups|vector_dbs` cli from the llama-stack-client repo. for consistency, this new command should go into `llama-stack-client`, but maybe we should consolidate under `llama stack ...`. thoughts?